integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala

```diff
@@ -48,16 +48,11 @@ object CompareResults {
         .config("spark.rapids.sql.enabled", "false")
```

Comment:
2021 copyrights need to be added to this file.

Reply:
added